search/enrich/enrich.md

```diff
@@ -0,0 +1,34 @@
+## Enrich
+
+The `enrich` module enables adding enumerated values to all entries in a pipeline within the query or using resources. `enrich` can be used to annotate data with constant enumerated values, such as "User=admin", in order to simplify visualization and reporting, pivoting within compound queries, and working with nontemporal data. 
```

Comment:
"enables adding" is clunky. what about "The `enrich` module can add enumerated values to each entry in a pipeline; these values can be specified in the module arguments or can come from a resource."

Reply:
done